api/registration/models.py

```diff
@@ -77,4 +85,52 @@ def password_reset_token_created(sender, instance, reset_password_token, *args,
     subject = 'Reset your password'
     message = f'Dear {reset_password_token.user.full_name},\n\nClick on the link below to reset your password.\n{absolute_url}\n\nCLIMAN Team\n2023'
     msg = EmailMessage(subject, message, to=[reset_password_token.user.email])
-    msg.send()
\ No newline at end of file
+    msg.send()
+
+
+class Appointments(models.Model):
+    # The primary key for the Appointments model
+    # is a UUID field that is generated automatically.
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    # The foreign key field that references the User model
+    # and represents the doctor who will be seeing the patient.
+    doctor = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='doctor_appointments')
+
+    # The foreign key field that references the User model
+    # and represents the patient who has scheduled the appointment.
+    patient = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='patient_appointments')
+
+    # A string field that stores the type of treatment or reason for the appointment.
+    treatment = models.CharField(max_length=100)
+
+    # A text field that stores any additional notes or details about the appointment.
+    description = models.TextField()
+
+    # A date field that stores the date of the appointment.
+
+    date = models.DateField(default=datetime.date.today)
+
+    # A timestamp field that stores the time of day when the appointment is scheduled to occur.
+    time = models.TimeField()
+
+    # A string field that stores the current status of the appointment.
+    # It has three possible values: 'SCHEDULED', 'DONE', or 'CANCELED'.
+    status = models.CharField(max_length=1, choices = AppointmentStatus.choices, default=AppointmentStatus.Scheduled)
+
+
```

Comment:
I don't get the rest of the code in this file
But I feel like it is unrelevant to the task itself

Reply:
Do you mean the code below or above?

Reply:
Below